### DISCUSSION TITLE
In the crop rotation sequence of Canola-Wheat-Canola-Wheat,

### DISCUSSION THREAD

**Discussion by cranju** [APSIMInitiative/ApsimX#9313] [Q&A]
Hi, 
I did run crop rotation in APSIM for two sequences:  R1: Canola-Wheat-Canola-Wheat & R2: Wheat- Canola-Wheat-Canola. It did run and overall, the yield for wheat is much higher than canola (almost 3 times). One contributing factor for this difference may be the planting density but need to investigate this further.  However, there is an issue with the rotation simulation. For some reason, for my rotation Canola-Wheat-Canola-Wheat, the model is taking Wheat first in the rotation and for the rotation Wheat-Canola-Wheat-Canola, the model is taking Canola first in the rotation. What might be the potential cause for this, given I am planning to run this for a long time historical climate and future climate? 
[Crop Rotation.zip](https://github.com/user-attachments/files/16972707/Crop.Rotation.zip)

**Comment by KeLiu7** [comment]
Hi Ranju, as I said Try to change the initial state from fallow to initial then you will get the right crop rotation order that you want.
